src/geometry/CMakeLists.txt

```
@@ -7,10 +7,30 @@ file(GLOB SOURCES "*.cpp")
 add_library(redisearch-geometry STATIC ${SOURCES})
 
 # include_directories(${absl_INCLUDE_DIR})
+message(STATUS "BOOST_DIR: ${BOOST_DIR}")
 
-if(NOT BOOST_DIR STREQUAL "")
+if(NOT BOOST_DIR STREQUAL "" AND EXISTS ${BOOST_DIR})
  	message(STATUS "BOOST_DIR is not empty: ${BOOST_DIR}")
 	include_directories(${BOOST_DIR})
 else()
- 	message(STATUS "BOOST_DIR is not defined")
+ 	message(STATUS "BOOST_DIR is not defined or empty")
```

Make it even clearer?
```suggestion
 	message(STATUS "BOOST_DIR is not defined or is empty")
```